docker-compose/start_dse_68.sh

```diff
@@ -37,4 +37,4 @@ export JSONTAG
 
 echo "Running with DSE $DSETAG, Stargate $SGTAG, JSON API $JSONTAG"
 
-docker-compose up -d --wait
```

Comment:
This has both its advantages and disadvantages. The advantage is that this can be simply used in any work flow (for it to wait before moving to next step) but the disadvantage is to have the latest docker version.
My take will be to remove '--wait', so we will be compatible with old versions too and also we can work it around in any work flow by adding something like below (this is some thing I added in a Github action post docker compose)

```
- name: Wait for services
  run: |
     while ! nc -z localhost 8080; do sleep 1; done
     while ! nc -z localhost 8081; do sleep 1; done
```

Reply:
I noticed, FWTW, that Docker desktop had "enable docker compose v2" so at least I was able to upgrade easily.

Reply:
I would remove it for now, we can add later on if needed and if used in any of the workflows.

Reply:
cc @jeffreyscarpenter